app/src/main/java/com/mozilla/hackathon/kiboko/services/SyncHelper.java

```diff
@@ -112,14 +109,13 @@ public boolean performSync(@Nullable SyncResult syncResult, Account account, Bun
      * the new data and import it into the database.
      *
      * @return Whether or not data was changed.
-     * @throws IOException if there is a problem downloading or importing the data.
      */
-    private boolean doDsoDataSync() throws IOException {
+    private boolean doDsoDataSync() {
         if (!isOnline()) {
             LOGD(TAG, "Not attempting remote sync because device is OFFLINE");
             return false;
         }
-
+        //TODO: Remove this function, it doesn't serve a purpose
```

Comment:
Yep, I just left it in for now in case there were any other comments that referenced it.